ztp/ztp-ran-policy-generator/sourcePolicies/ClusterLogSubscription.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging
+spec:
+  channel: "5.0"
+  name: cluster-logging
+  source: redhat-operators
```

Comment:
A general comment, how are we overridig the source for disconnected deployments ?